axe_selenium_python/axe.py

```diff
@@ -33,7 +33,8 @@ def execute(self, context=None, options=None):
         :param context: which page part(s) to analyze and/or what to exclude.
         :param options: dictionary of aXe options.
         """
-        template = "return axe.run(%s).then(function(result){return result;});"
+        template = "var callback = arguments[arguments.length - 1];" \
+                   "axe.run(%s).then(results => callback(results))"
```

Comment:
Awesome work :)

I don't have a lot of experience with JavaScript; I tried a lot of different approaches and didn't get it to work!